### PR TITLE
Axel dubar/feature/glossary

### PR DESCRIPTION
**Pull request by AxelDubar**
Glossary page which is accessible at https://peerannot.github.io/glossary/ 
(or from the toolbar)

### CODE REVIEW

**Comment by josephsalmon** [comment]
LGTM. @tanglef what about you?

**Comment by tanglef** [comment]
> LGTM. @tanglef what about you?

@josephsalmon we had a discussion about adding the mathematical definition of labels (hard vs soft, class number vs vector in the simplex)  and then it's good for me too

File: _posts/2023-10-24-glossary.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+| $y^{(j)}_i$             | The answer of the worker $j$ to the task $i$                          |                                                                                                                                                      |
+| K                       | The set of labels a task can take                                     |                                                                                                                                                      |
+| AccTrain($\mathcal{D}$) | A metric that measure aggregation strategies' accuracies              | AccTrain($\mathcal{D}$) = $ \frac{1}{\|\mathcal{D}\|} \sum\_{i=1}^{\|\mathcal{D}\|} \unicode{x1D7D9}\_{y\_i = \text{argmax}_{k\in \[K\]}(ŷ\_i)\_k} $ |
+
+## Commands
+
+| Name           | Definition                                                                                  |
+| -------------- | ------------------------------------------------------------------------------------------- |
+| identify       | Command that include the methods allowing to indentify difficult tasks or mislabeled labels |
```
Comment (tanglef) [inline _posts/2023-10-24-glossary.md]: difficult and/or mislabeled tasks and worker abilities

File: _posts/2023-10-24-glossary.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| aggregate      | Command that include the methods allowing to aggregate label drom common noise model        |
```
Comment (tanglef) [inline _posts/2023-10-24-glossary.md]: label from common noise model

File: _posts/2023-10-24-glossary.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+| aggregate-deep | Command that include the methods that train a model directly from the non aggregated data   |
+| train          | Command that include the methods that train a model using aggregated data                   |
```
Comment (tanglef) [inline _posts/2023-10-24-glossary.md]: aggregated labels

File: _posts/2023-10-24-glossary.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+| K                       | The set of labels a task can take                                     |                                                                                                                                                      |
+| AccTrain($\mathcal{D}$) | A metric that measure aggregation strategies' accuracies              | AccTrain($\mathcal{D}$) = $ \frac{1}{\|\mathcal{D}\|} \sum\_{i=1}^{\|\mathcal{D}\|} \unicode{x1D7D9}\_{y\_i = \text{argmax}_{k\in \[K\]}(ŷ\_i)\_k} $ |
+
+## Commands
+
+| Name           | Definition                                                                                  |
+| -------------- | ------------------------------------------------------------------------------------------- |
+| identify       | Command that include the methods allowing to indentify difficult tasks or mislabeled labels |
+| aggregate      | Command that include the methods allowing to aggregate label drom common noise model        |
+| aggregate-deep | Command that include the methods that train a model directly from the non aggregated data   |
```
Comment (tanglef) [inline _posts/2023-10-24-glossary.md]: from the raw collection of labels

File: _posts/2023-10-24-glossary.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+| $\mathcal{Lab(x\_i)}$   | The vector of answered labels of the task $i$                         | $(y_i^{(j)})_{j\in\mathcal{A(x\_i)}}$                                                                                                                |
+| $y_i^*$                 | The true label of the task $i$                                        |                                                                                                                                                      |
+| $ŷ_i^{agg}$             | The computed label of the task $i$ given the aggregation $agg$ method |                                                                                                                                                      |
+| $\pi^{(j)}$             | The confusion matrix of the worker $j$                                |                                                                                                                                                      |
```
Comment (tanglef) [inline _posts/2023-10-24-glossary.md]: Mathematical definition: \pi^{(j)}_{k,\ell}=\mathbb{P}(y_i^{(j)​}=\ell∣y_i^\star​=k) \forall (\ell,k)\in [K]^2

File: _posts/2023-10-24-glossary.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+| $y^{(j)}_i$             | The answer of the worker $j$ to the task $i$                          |                                                                                                                                                      |
+| K                       | The set of labels a task can take                                     |                                                                                                                                                      |
```
Comment (tanglef) [inline _posts/2023-10-24-glossary.md]: [K] = \{1,...,K\} is the set, K alone is the number of possible labels

**Review by tanglef — CHANGES_REQUESTED** [review]
Some notes on the definitions

**Comment by tanglef** [comment]
All good for me thanks !

**Review by tanglef — APPROVED** [review]

**Pull request merged**